newsfragments/9109.bugfix.rst

```diff
@@ -0,0 +1 @@
+[CLI] Stop spinners with check mark.
```

Comment:
The newsfragment must use RST syntax, so you can't have this `[CLI]` marker.

See https://docs.parsec.cloud/en/stable/history.html for reference.